desktop/apps/artwork/components/artists/query.coffee

```diff
@@ -1,6 +1,7 @@
 module.exports = """
   fragment artists on Artwork {
     artists {
+      _id
```

Comment:
Since we want to store artist id in the DOM (for use client-side by the React market insights component), we add it to the query.